extensions/amp-story/1.0/amp-story-reaction-poll.css

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.i-amphtml-story-reaction-poll-container {
+  display: flex !important;
+  justify-content: center !important;
+  height: 90px !important;
+  max-width: 296px !important;
+  border-radius: 16px !important;
+  box-shadow: 0px 2px 4px #000000aa !important;
```

Comment:
Nit: rgba(0, 0, 0, 0.12) for the color as it's what we use in most other places

Reply:
Sure! If all the reaction components get a shadow we might move this to the reaction.css, but here for now since quizzes don't have shadows AFAIK